db/migrate/20170315152028_create_conversations.rb

```diff
@@ -0,0 +1,11 @@
+class CreateConversations < ActiveRecord::Migration[5.0]
+  def change
+    create_table :conversations do |t|
+      t.string :content, null: false
+      t.references :comment, null: false
+      t.references :user, null: false
```

Comment:
Looks like each `conversation` belongs to a specific `comment` and a specific `user`.  The specs asked for conversations on movies. (So a conversation would have many comments, but each conversation would belong to one movie.)